crm_action/data/email_reminder.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<openerp>
+
+<data noupdate="1"> <!-- noupdate = 1 for the 'active' field -->
```

Comment:
no need to state the reason for noupdate. It's never a good idea anyway to have noupdate="0" for crons and email templates.